00-HTML-CSS-basics/ex 2/2.2/css/style.css

```diff
@@ -0,0 +1,18 @@
+html{
+    font-family: "Times New Roman", Times, serif;
+    font-size:14px;
```

Comment:
should leave an space before the 14px, this for all css attributes like:

`background-color: red;`